src/components/TouchableWithoutFocus.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {TouchableOpacity} from 'react-native';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    /** Element that should be clickable  */
+    children: PropTypes.oneOfType([
+        PropTypes.node,
+        PropTypes.func,
+    ]).isRequired,
+
+    /** Callback for onPress event */
+    onPress: PropTypes.func.isRequired,
+
+    /** Styles that should be passed to touchable container */
+    styles: PropTypes.arrayOf(PropTypes.object),
+};
+
+const defaultProps = {
+    styles: [],
+};
+
+/**
+ * This component prevents the tapped element from capturing focus
+ */
+class TouchableWithoutFocus extends React.Component {
+    constructor(props) {
+        super(props);
+        this.onPress = this.onPress.bind(this);
+    }
+
+    onPress() {
+        if (!this.props.onPress) {
```

Comment:
`onPress` is required in the props, so I don't think this early return is useful.

Reply:
Removed the early return